rails/spec/requests/api/public_stories_spec.rb

```diff
@@ -127,5 +164,14 @@ def json_response
       expect(json_response["stories"].map { |s| s["id"] }).to contain_exactly(story_1.id, story_2.id)
 
     end
+
+    it "orphaned stories are excluded" do
+      speaker_3.destroy
+      place_3.destroy
+
+      get "/api/communities/orphaned_community/stories"
+      expect(json_response["total"]).to eq(0)
+      expect(json_response["stories"].length).to eq(0)
+    end
```

Comment:
Could be worthwhile to also test for a Story to not show if it has 0 Speakers but >1 Places, and 0 Places but >1 Speakers.

Reply:
Hey @rudokemper! This is actually what I did with how the stories are setup, but the numbered speaker and place don't make it very obvious, I'll update the variable names for clarity.